src/main/java/site/coach_coach/coach_coach_server/coach/domain/Coach.java

```diff
@@ -0,0 +1,59 @@
+package site.coach_coach.coach_coach_server.coach.domain;
+
+import java.time.LocalDateTime;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import site.coach_coach.coach_coach_server.user.domain.User;
+
+@Table(name = "coaches")
+@Entity
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Coach {
```

Comment:
아래와 같이 해주시면 따로 createdAt을 지정안해주셔도 됩니당!
`import site.coach_coach.coach_coach_server.common.domain.DateEntity;`해주세요!

```suggestion
public class Coach extends DateEntity {
```